rancher/installing-rancher/installing-server/index.md

```diff
@@ -30,6 +30,16 @@ Once the UI is up and running, you can start [adding hosts]({{site.baseurl}}/ran
 
 <a id="external-db"></a>
 
```

Comment:
Can you move this so that it's above the <a id="external-db"></a>? I use that tag to link from other pages so it needs to be right above the instructions for external DB.